docs/release/1.28/release-note_1.28.0.md

```diff
@@ -0,0 +1,43 @@
+# Release Note 1.28.0
+
+## ONE Runtime
+
+### Python API
+
+- Support experimental python API
+  - Refer howto document for more details
+
+### On-device Training
+
+- Support on-device training with circle model
+  - Training parameter can be passed to _onert_ via _onert_`s experimental API or loading new model format including training information: _circle_plus_
+  - Trained model can be exported to _circle_ model via experimental API `nnfw_train_export_circle`
+  - Supporting Transfer learning from a pre-trained circle model
```

Comment:
(optional) It would be good to add reference to https://github.com/Samsung/ONE/blob/master/docs/runtime/transfer_learning.md.